src/typography/p5.Font.js

```diff
@@ -475,7 +475,8 @@ p5.Font = class {
   _handleAlignment(renderer, line, x, y, textWidth) {
     const fontSize = renderer._textSize;
 
-    if (typeof textWidth === 'undefined') {
+    if (typeof textWidth === 'undefined' &&
+    renderer._textAlign !== constants.LEFT) {
```

Comment:
What is the rendering issue you are mitigating here?

Reply:
Correct me if I'm wrong @munusshih but I believe this is just avoiding the textWidth calculation if we don't need the `textWidth` value 🙂 The variable is only used below if we have CENTER or RIGHT alignment. There's no functional change here.